app/Http/Controllers/ContactsController.php

```diff
@@ -74,6 +82,18 @@ public function index(Request $request)
             $contacts = $user->account->contacts()->real()->sortedBy($sort)->get();
         }
 
+        if ($date_flag) {
+            foreach ($contacts as $contact) {
+                $contact['sort_date'] = $contact->getLastActivityDate();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "sort_date" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1192)